dbt/models/default/default.vw_pin_sale.sql

```diff
@@ -249,93 +244,166 @@ sales_val AS (
         sf.run_id AS sv_run_id,
         sf.version AS sv_version
     FROM
-        {{ source('sale', 'flag') }}
-            AS sf
+        {{ source('sale', 'flag') }} AS sf
     INNER JOIN max_version_flag AS mv
         ON sf.meta_sale_document_num = mv.meta_sale_document_num
         AND sf.version = mv.max_version
+),
+
+combined_sales AS (
+    SELECT
+        COALESCE(uq_sales.pin, md_sales.pin) AS pin,
+        COALESCE(uq_sales.year, md_sales.year) AS year,
+        COALESCE(uq_sales.township_code, tc.township_code) AS township_code,
+        COALESCE(uq_sales.nbhd, tc.nbhd) AS nbhd,
+        COALESCE(uq_sales.class, tc.class) AS class,
+        COALESCE(uq_sales.sale_date, md_sales.sale_date) AS sale_date,
+        COALESCE(uq_sales.sale_price, md_sales.sale_price) AS sale_price,
+        uq_sales.sale_key,
+        COALESCE(uq_sales.doc_no, md_sales.doc_no) AS doc_no,
+        COALESCE(uq_sales.deed_type, md_sales.mydec_deed_type) AS deed_type,
+        COALESCE(uq_sales.seller_name, md_sales.seller_name) AS seller_name,
+        COALESCE(uq_sales.is_multisale, md_sales.is_multisale) AS is_multisale,
+        COALESCE(uq_sales.num_parcels_sale, md_sales.num_parcels_sale) --noqa
+            AS num_parcels_sale,
+        COALESCE(uq_sales.buyer_name, md_sales.buyer_name) AS buyer_name,
+        COALESCE(uq_sales.sale_type, NULL) AS sale_type,
```

Comment:
Any idea if there is a mydec equivalent for this? I looked but it wasn't obvious to me